pantalaimon/daemon.py

```diff
@@ -581,7 +581,8 @@ async def start_pan_client(self, access_token, user, user_id, password, device_i
 
         if password == "":
             if device_id is None:
-                raise ValueError("Empty password provided, but device_id was also None")
+                logger.warn(f"Empty password provided and device_id was also None")
```

Comment:
Sorry to be annoying but why did this turn into an f-string, also we should explain what the ramifications of that error are.
```suggestion
                logger.warn(
                    "Empty password provided and device_id was also None, not "
                    "starting background sync client "
                )
```

Reply:
copy paste fail. Thanks for fixing.